apps/mason2/CMakeLists.txt

```diff
@@ -26,10 +26,10 @@ if (NOT ZLIB_FOUND)
   return ()
 endif (NOT ZLIB_FOUND)
 
-if (MSVC)
-  message (STATUS "  Not building MASON2 under Visual Studio.")
-  return ()
-endif()
+# Mason2 does not work correctly with Visual Studio Version <10 on 64 bit systems.
+if (MSVC AND MSVC_VERSION LESS 1600)
```

Comment:
I think you won't need this.
For the entire SeqAn repo we require at least VS15 to get full c++11 support.